src/main/java/org/prebid/server/handler/openrtb2/AmpHandler.java

```diff
@@ -226,6 +229,17 @@ private static ExtResponseDebug extResponseDebugFrom(BidResponse bidResponse) {
         return extBidResponse != null ? extBidResponse.getDebug() : null;
     }
 
+    private static Map<String, List<BidderErrorInfo>> errorsFrom(BidResponse bidResponse) {
+        final ExtBidResponse extBidResponse;
+        try {
+            extBidResponse = Json.mapper.convertValue(bidResponse.getExt(), EXT_BID_RESPONSE_TYPE_REFERENCE);
```

Comment:
It's not efficient to deserialize the extension two times (here and in `extResponseDebugFrom`), one time should be enough